test/automated/ansible/roles/install-gpg/tasks/main.yml

```diff
@@ -1,9 +1,10 @@
 ---
 
 - name: 'install gpg'
-  ansible.builtin.package:
-    name: gpgv
+  apt:
```

Comment:
is this only for debian(ubuntu) based images?

Reply:
only for debian buster yep